javascript/javascript1/week1/housey-pricey.js

```diff
@@ -0,0 +1,28 @@
+
+//Peter case
+let width = 8
+let height = 10
+let depth = 10
+let gardenSizeInM2 = 100
+let paid = 2500000
+
+//formulas
+let volumeInMeters = depth * height * width;
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+
+//result
+console.log(`Peter's house price should be ${housePrice}, and he paid ${paid}`);
+
+//Julia case
+width = 5
+height = 8
+depth = 11
+gardenSizeInM2 = 70
+paid = 1000000
+
+//formulas
+volumeInMeters = depth * height * width;
+housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+
+//result
+console.log(`Julia's house price should be ${housePrice}, and she paid ${paid}`);
```

Comment:
It correctly outputs the results using the formular, but it does not evalutate whether the person, should or shouldn't purchase the house.

I would add a line at the end: '[Name] should purchase the house: ${shouldPurchase}'. Where should purchase is boolean value.